### PR TITLE
Revert "Revert "feat(troubleshoot): add filesystem latency check""

### PR DESCRIPTION
**Pull request by emosbaugh**
Reverts replicatedhq/embedded-cluster#1024

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -323,6 +323,36 @@ jobs:
         with:
           message-path: download-link.txt
 
+  e2e-light:
```
Comment (laverya) [inline .github/workflows/ci.yaml:326]: is this just e2e tests run inside a docker container?
If so, a comment would be great

**Review by laverya — APPROVED** [review]

File: pkg/preflights/host-preflight.yaml [hunk 2]
```diff
@@ -367,3 +375,12 @@ spec:
...
               message: "resolv.conf does not reference '127.0.0.1'"
+    - filesystemPerformance:
+        checkName: Filesystem Write Latency
+        collectorName: filesystem-write-latency-etcd
+        outcomes:
+          - pass:
+              when: "p99 < 10ms"
+              message: 'Write latency is ok (p99 target < 10ms, actual: {{ "{{" }} .P99 {{ "}}" }})'
+          - fail:
+              message: 'Write latency is high (p99 target < 10ms, actual: {{ "{{" }} .P99 {{ "}}" }})'
```
Comment (ajp-io) [inline pkg/preflights/host-preflight.yaml:386]:
```suggestion
          - pass:
              when: "p99 < 10ms"
              message: 'P99 write latency for the disk at /var/lib/k0s/etcd is {{ "{{" }} .P99 {{ "}}" }}, which is better than the 10 ms requirement.'
          - fail:
              message: 'P99 write latency for the disk at /var/lib/k0s/etcd is {{ "{{" }} .P99 {{ "}}" }}, but it must be less than 10 ms. A higher-performance disk is required.'
```

**Review by laverya — APPROVED** [review]

File: pkg/preflights/host-preflight.yaml [hunk 3]
```diff
@@ -355,4 +363,24 @@ spec:
               message: "Neither 'nameserver localhost' nor 'nameserver 127.0.0.1' can be present in resolv.conf. Remove them to continue."
           - pass:
               when: "false"
-              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
\ No newline at end of file
+              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
```
Comment (ajp-io) [inline pkg/preflights/host-preflight.yaml:366]: This seems to have replaced my changes to merge these two resolv.conf preflights into one. Copy and paste from here to fix it. https://github.com/replicatedhq/embedded-cluster/blob/main/pkg/preflights/host-preflight.yaml#L348-L358

File: pkg/preflights/host-preflight.yaml [hunk 4]
```diff
@@ -355,4 +363,24 @@ spec:
...
+        checkName: 127.0.0.1 in resolv.conf
+        fileName: host-collectors/run-host/resolv.conf.txt
+        regex: 'nameserver 127.0.0.1'
+        outcomes:
+          - fail:
+              when: "true"
+              message: "resolv.conf should not reference '127.0.0.1', this will prevent coredns within the cluster from resolving"
+          - pass:
+              when: "false"
+              message: "resolv.conf does not reference '127.0.0.1'"
```
Comment (ajp-io) [inline pkg/preflights/host-preflight.yaml:377]: The two resolv.conf preflights were merged into one, which it looks like is above, so this one can be removed.
```suggestion
```

**Review by laverya — APPROVED** [review]

**Pull request merged**